plugins/modules/monitoring/nagios.py

```diff
@@ -358,7 +395,7 @@ def __init__(self, module, **kwargs):
         self.comment = kwargs['comment']
         self.host = kwargs['host']
         self.servicegroup = kwargs['servicegroup']
-        self.start = int(kwargs['start'])
+        self.start = kwargs['start']
```

Comment:
Why did you remove the `int()`? This way, you end up with a string.

(The module's argument handling needs an overhaul anyway, most types are not specified. But that's for another PR...)

Reply:
because it is an optional parameter.
When it is not provided the module throws an 'cannot convert NullType' error

Reply:
But in that case, you need to only use `int()` if it is not `None`. Completely removing it can potentially break parts of the module which rely on this being an int.